linode/manager
# DBAAS1-80: Database Detail Data Scaffolding

## Description

- Sources data for Database Details and Summary with React Query
- Following PRs for DBaaS are bound to make changes to this initial scaffolding
- Fixed an incorrect type in `@linode/api-v4`
- Added **MSW** mock for `/databases/:engine/instances/:id`

## How to test

- Go to a Database details page (`/databases/3` for example) and make sure you see JSON data.

<img width="556" alt="Screen Shot 2021-12-22 at 12 05 50 PM" src="https://user-images.githubusercontent.com/6440455/147129268-4748dff5-906e-4435-b687-1d1e74ead561.png">
